### ISSUE TITLE
run with big data breakdown sometimes

### ISSUE THREAD

**Issue by mengfanyu1991** [tensorflow/tensorflow#21110]
I use .pb file to prediction . if I use sample image is good .  if the I use big data，sometime while breakdown.   I use  tensorflow1.3 1.4 1.5 is OK,  but the version is higher than 1.5 have the problem.The pb is producted with tensorflow 1.4. 

CUB segmented reduce errorinvalid configuration argument
   [[Node: inception_v3/logits/predictions = Softmax[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"](inception_v3/logits/logits/xw_plus_b)]]

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by cy89** [comment]
@mengfanyu1991 could you please fill in the form, and give us enough data to be able to reproduce your problem?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 44 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by dksb** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
